plugins/azure/resoto_plugin_azure/resource/containerservice.py

```diff
@@ -0,0 +1,975 @@
+
+
+@define(eq=False, slots=False)
+class AzureUserAssignedIdentity:
+    kind: ClassVar[str] = "azure_user_assigned_identity"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "client_id": S("clientId"),
+        "object_id": S("objectId"),
+        "resource_id": S("resourceId"),
+    }
+    client_id: Optional[str] = field(default=None, metadata={'description': 'The client ID of the user assigned identity.'})  # fmt: skip
+    object_id: Optional[str] = field(default=None, metadata={'description': 'The object ID of the user assigned identity.'})  # fmt: skip
+    resource_id: Optional[str] = field(default=None, metadata={'description': 'The resource ID of the user assigned identity.'})  # fmt: skip
+
+
+@define(eq=False, slots=False)
+class AzureManagedServiceIdentity:
+    kind: ClassVar[str] = "azure_managed_service_identity"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "principal_id": S("principalId"),
+        "tenant_id": S("tenantId"),
+        "type": S("type"),
+        "user_assigned_identities": S("userAssignedIdentities"),
+    }
+    principal_id: Optional[str] = field(default=None, metadata={'description': 'The service principal ID of the system assigned identity. This property will only be provided for a system assigned identity.'})  # fmt: skip
+    tenant_id: Optional[str] = field(default=None, metadata={'description': 'The tenant ID of the system assigned identity. This property will only be provided for a system assigned identity.'})  # fmt: skip
+    type: Optional[str] = field(default=None, metadata={'description': 'Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).'})  # fmt: skip
+    user_assigned_identities: Optional[Dict[str, AzureUserAssignedIdentity]] = field(default=None, metadata={'description': 'The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.'})  # fmt: skip
+
+
+@define(eq=False, slots=False)
+class AzureFleet(AzureResource):
+    kind: ClassVar[str] = "azure_fleet"
+    api_spec: ClassVar[AzureApiSpec] = AzureApiSpec(
+        service="containerservice",
+        version="2023-08-15-preview",
+        path="/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/fleets",
+        path_parameters=["subscriptionId"],
+        query_parameters=["api-version"],
+        access_path="value",
+        expect_array=True,
+    )
+    mapping: ClassVar[Dict[str, Bender]] = AzureTrackedResource.mapping | {
+        "id": S("id"),
+        "tags": S("tags", default={}),
+        "name": S("name"),
+        "ctime": K(None),
+        "mtime": K(None),
+        "atime": K(None),
```

Comment:
can you walk the list of ctime, atime, mtime props - either define or delete them.